snuba/snuba_migrations/events/0018_errors_ro_add_tags_hash_map.py

```diff
@@ -0,0 +1,62 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import UUID, Array, Column, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.storages.tags_hash_map import TAGS_HASH_MAP_COLUMN
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = True
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.EVENTS,
```

Comment:
should these StorageSetKey's be `StorageSetKey.EVENTS_RO` instead?

Reply:
fixed